.github/workflows/bank-api-library.check.yml

```diff
@@ -71,15 +75,4 @@ jobs:
         CODE_SIGNING_REQUIRED=NO
         ONLY_ACTIVE_ARCH=NO
         CLIENT_ID="gini-mobile-test"
-        CLIENT_SECRET="${{ secrets.GINI_MOBILE_TEST_CLIENT_SECRET }}"
```

Comment:
@GeraltRiv why was it removed it's a part of pinning tests?

Reply:
we need to run 1 test separately for  PinningWrongCertificates

Reply:
Because it is 2 the same `xcodebuild clean test`
I think we just build and test it 2 times